plugins/idea/pluginBase/src/main/java/com/imaginea/kodebeagle/base/model/SearchQuery.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.imaginea.kodebeagle.base.model;
+
+import com.google.gson.Gson;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by bipulk on 10/8/16.
+ */
+public class SearchQuery {
+
+
+    private Set<QueryParam> queries = new HashSet<>();
+    private int from = 0;
+    private int size = 10;
+
+
+    public String build() {
+
+        Gson gson = new Gson();
+        String query = gson.toJson(this);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "query". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1488)